src/main/java/com/pinterest/secor/util/AvroSchemaUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.pinterest.secor.util;
+
+import com.pinterest.secor.common.SecorConfig;
+
+public class AvroSchemaUtil {
+
+    public static String getAvroSubjectSuffix(SecorConfig config) {
+        return config.getString("avro.schema.subject.suffix", "");
+    }
+
+    public static String getAvroSubjectOverride(SecorConfig config) {
```

Comment:
Is this a global override or per-topic override?

Reply:
This is a global override and has marked comments in the properties file:

```
# Setting this will override the subject lookup for all topics. This is only recommended if
# you are reading a single kafka topic.
avro.schema.subject.override=
```